### ISSUE TITLE
Blade exception for text

### ISSUE THREAD

**Issue by mxts** [laravel/framework#45424]
- Laravel Version: 9.1.4
- PHP Version: 8.1.13
- MySQL version: 8.0.31-0ubuntu0.22.04.1
- MySQL Client version: mysqlnd 8.1.13

### Description:
```
@include('subview', [
    'test' => '(',
])

// subview.blade.php
{{ $test }}
```
Blade above produces the error `ParseError syntax error, unexpected token ","`, the cause being the unclosed `(` inside the text.

While I understand this is an edge case, the exception does not point directly to the problem and the solution can be quite hard to spot. In my case, the offending text was a frowning face `:(`

### Steps To Reproduce:

I do have a repository, but this should be quite straightforward to reproduce. Thank you.

**Comment by bert-w** [comment]
https://github.com/laravel/framework/issues/18317

**Comment by mxts** [comment]
Thanks, perhaps it helps to hint about limitations in the subview documentation? I could file a PR for that.

**Comment by driesvints** [comment]
Looks like we won't be fixing this. Feel free to attempt to PR the docs 👍

**Comment by imanghafoori1** [comment]
This can be considered to be fixed as of version 9.47.0 or above.
@mxts
